Quran/QuranPageCollectionViewCell.swift

```diff
@@ -11,7 +11,12 @@ import AVFoundation
 
 private let imageHeightDiff: CGFloat = 34
 
-class QuranPageCollectionViewCell: UICollectionViewCell {
+@objc protocol QuranPageCollectionCellDelegate {
```

Comment:
Same thing for this protocol, I think it could be just a class protocol no need for `@objc`, what do you think?